src/common/SubProcess.h

```diff
@@ -99,6 +104,10 @@ class SubProcess {
   int stderr_pipe_in_fd;
   int pid;
   std::ostringstream errstr;
+
+  #ifdef _WIN32
+  HANDLE handle;
```

Comment:
Nit: just initialize it to `NULL` here to avoid the constructor initialization (and associated guard)